enterprise/models.py

```diff
@@ -322,6 +334,56 @@ def __repr__(self):
         return self.__str__()
 
 
+@python_2_unicode_compatible
+class EnterpriseCustomerIdentityProvider(TimeStampedModel):
+    """
+    EnterpriseCustomerIdentityProvider is a One to One relationship between Enterprise Customer and Identity Provider.
+
+    There should be a link between an enterprise customer and its Identity Provider. This relationship has
+    following constraints
+        1. An enterprise customer may or may not have an identity provider.
+        2. An enterprise customer can not have more than one identity providers.
+        3. Enterprise customer site should match with identity provider's site. (i.e. same domain names)
+
+    Fields:
+        enterprise_customer (ForeignKey[EnterpriseCustomer]): enterprise customer
+        provider_id (:class:`django.db.models.PositiveIntegerField`): positive integer id of identity provider.
```

Comment:
Same here - it's a `SlugField`, not a `PositiveIntegerField`. And I think the description should be "The provider_id string of the identity provider".